### ISSUE TITLE
How do I disable Visual Mode? 

### ISSUE THREAD

**Issue by yanmuq** [seerge/g-helper#2297]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information will be provided, my issue will be closed without an answer.

### Is your feature request related to a problem? Please describe

I accidentally clicked to download the Visual Mode feature, which caused my screen color to change.

### Describe the solution you'd like

 I would like to be able to uninstall this feature on my computer.
At the same time, I wanted to be able to hide the button to download this feature in a deeper menu so that I didn't accidentally touch it. 

### Device and Model

rog gu603-hm

### Additional information.

![Uploading capture_20240326225930015.jpg…]()

**Comment by seerge** [comment]
@yanmuq hello, you just set Visual Mode to default which is equivalent to "no mode" selected. There is nothing to uninstall here :) 

Thanks.

**Comment by yanmuq** [comment]
The system I use is completely installed from Windows mirror. And Armory Crate has not been installed. In other words, my computer should not have AsusSplendid. But this setting still changes my screen color.

**Comment by seerge** [comment]
@yanmuq AsusSplendid is part of "Asus System Control Interface" driver as I told, that one is installed anyway.

If you don't like the factory calibrated color profile coming from Asus, you can also completely turn it off from Windows ``Color Management`` app.

**Comment by yanmuq** [comment]
I don't want the AsusSplendid program. I want to use win's color management. Currently I cannot change the color in the win settings interface. The brand color does not change when using different color profiles.

**Comment by yanmuq** [comment]
screen

**Comment by seerge** [comment]
No one is forcing you to use that. Just set mode to ``Default` and you are good to go.

**Comment by yanmuq** [comment]
But I can't change the color profile of the screen right now, the screen color doesn't change after changing it

**Comment by seerge** [comment]
@yanmuq it's very hard to understand what you are trying to do. Also you don't seem to read what I'm answering.

If you are already on Default mode of course it won't change if you just click on same Default again.

**Comment by yanmuq** [comment]
Originally, I was using a color profile generated by a calibrator instead of a color file that came out

**Comment by yanmuq** [comment]
I chose a different color profile in the settings - screen interface of win10 and it didn't work anymore

**Comment by seerge** [comment]
@yanmuq if you were using a 3rd party color profile, you can manually assign it to the screen at windows ``Color Management`` application as I have told you. Next to that make sure that you reset all color settings in ``Intel Graphics Command Center`` to defaults.

**Comment by yanmuq** [comment]
Originally, the screen color would change as I selected different files

**Comment by yanmuq** [comment]
If I don't have AsusSplendid originally, will it be installed automatically. And if I don't want it, I can uninstall it and how to uninstall it

**Comment by seerge** [comment]
@yanmuq it's installed by windows update as part of the driver, as I have mentioned above.

However, unless you manually change Visual Modes in the app, it's not being used.

**Comment by seerge** [comment]
@yanmuq you can check build from this issue (and follow my advice there) to completely hide it from UI
https://github.com/seerge/g-helper/issues/2307

**Comment by seerge** [comment]
@yanmuq  PS also you can turn OFF this setting in windows https://github.com/seerge/g-helper/wiki/FAQ#my-display-colors-seem-off-even-if-i-set-visual-mode-to-default

**Comment by seerge** [comment]
@yanmuq @User198263321 hello,

I have recently discovered how Armoury can completely disable Game Visual by calling `AsusSplendid.exe` (responsible for it) and added it to G-Helper as new "Disabled" Visual Mode.

It is live in latest release https://github.com/seerge/g-helper/releases/tag/v0.196

Let me know if it changes something for you

**Comment by yanmuq** [comment]
Hi,
This update is extremely useful to me and has resolved the issue that was bothering me, even though the problem originated from the OEM. Thank you for addressing this problem!

**Comment by IamNishanKhan** [comment]
Same thing happened to me also recently this month. I mistakenly pressed `FN + V` shortcut which changed the display visual mode, then I pressed `FN + V` more till the color became default. But something was off but I was too busy to figure it out what has happened. One day one of my colleague told me that my laptop's display sucks, then I realized that display color looks like it has been washed off. I pressed `FN + V` many times but wasn't able to make it like before. I tried Default mode, I tried Disabled mode, none of them made it like it was before. 
I though about clean installing Windows but then figured that maybe display drivers can fix this, so I opened the `Intel® Graphics Command Center`, then went to Display > Color > Restore to Original Colors. After reset, the color become normal like before. 
Such a terrible and messy option tbh, made me suffer a lot. ASUS or GHelper don't know who to blame.
